backend/lib/mapping/BUILD

```diff
@@ -19,6 +19,17 @@ java_library(
     deps = lib_deps,
 )
 
+java_binary(
```

Comment:
@lucapette and @paulodiniz as long as we don't want to create ts definitions for anything else (which I think makes sense, because this problem is so unique), I would keep the generator as a binary that is local to this package.